packages/neuron-wallet/src/block-sync-renderer/tx-status-listener.ts

```diff
@@ -33,6 +35,13 @@ const getTransactionStatus = async (hash: string) => {
       blockHash: txWithStatus.txStatus.blockHash,
     }
   }
+  if (txWithStatus.txStatus.isRejected()) {
+    return {
+      status: TransactionStatus.Failed,
```

Comment:
Why not add a `Reject` status for `status` to replace adding a variable named `isRejected`?